### ISSUE TITLE
DOI

### ISSUE THREAD

**Issue by scls19fr** [scls19fr/windrose#54]
Publishing to Journal of Open Source Software (JOSS) http://joss.theoj.org/ may be consider to get a DOI (Digital Object Identifier)

**Comment by scls19fr** [comment]
Some researchers are using windrose. So @LionelR what is your opinion about publishing to Journal of Open Source Software?

**Comment by LionelR** [comment]
Bonjour Sébastien,
Je viens juste de réaliser que d'autres mails sont passés à la trappe par mon ancien anti-spam, désolé donc de n'avoir pas répondu aux autres messages. Je suis épaté par ce qui est fait sur ce module et ton travail. Tu as toute latitude pour publier sur le JOSS

**Comment by scls19fr** [comment]
Ok. Il faudrait par contre que tu te crées un compte sur https://orcid.org/
et que tu me confirmes l'affiliation que tu voudrais voir y figurer.

Il est possible de créer le compte orcid en se connectant sur http://joss.theoj.org/

**Comment by LionelR** [comment]
orcid.org/0000-0001-6017-4385

**Comment by scls19fr** [comment]
Submited
http://joss.theoj.org/papers/1b3fb0a7777ba65071e4d99b65a8c47d

**Comment by scls19fr** [comment]
PRE REVIEW
https://github.com/openjournals/joss-reviews/issues/263

**Comment by scls19fr** [comment]
REVIEW
https://github.com/openjournals/joss-reviews/issues/268

**Comment by scls19fr** [comment]
From https://github.com/openjournals/joss-reviews/issues/268#issuecomment-318530317
Zenodo can be use to get a DOI
http://blog.zenodo.org/2017/05/30/doi-versioning-launched/

I will add a new tag in the next days (CI is now fixed)

**Comment by scls19fr** [comment]
Digital Object Identifier is available at https://zenodo.org/account/settings/github/repository/python-windrose/windrose

**Comment by scls19fr** [comment]
I'm currently working on the paper submission for "The Journal of Open Source Software" JOSS for Python Windrose library (#77 )

See
https://github.com/python-windrose/windrose/blob/master/paper.md

and

https://github.com/python-windrose/windrose/blob/master/paper.bib

Now, paper.md have a paragraph named "List of contributors and/or notable users"

The following contributors are enumerated @LionelR , @scls19fr, @kilojoules , @xmnlab , @weber-s , @fmaussion , @ocefpaf because they appears on https://github.com/python-windrose/windrose/graphs/contributors

Could you please submit a PR to fix and/or add affiliation next to your name ? (or just add a comment here)

Any contribution about how you are using / how you were using Python Windrose is also welcome.
Just add some comments here...

I would like to also ping users of this library who opened an issue helping improvement of this library...
Feel free to add you name and affiliation also in this paragraph or comment here if you think it can help.

I was looking for an automatic way of doing this... https://stackoverflow.com/questions/50289171/how-to-retrieve-the-list-of-all-users-that-opened-an-issue-or-add-comment-to-an

but until I have no answer to this SO question I will ping python-windrose users who contributed manually...

@adityasinghwa @jankatins @Data-drone @cqcn1991 @joddm @slharris @brunorpinho @jtmcleod1989 @wanglongqi @georgebv @NicWayand @blaylockbk @loneharoon @daniclaar @dda-voisin @dennydengler @julietbravo @jsignell @mccannjb @ruffsl @Round-Walnut

Sorry for noise for those who lose interest in [python-windrose](https://github.com/python-windrose/windrose/).

Sorry mainly for those I forget... it's not personal ;-)

**Comment by fmaussion** [comment]
Hi, good idea with the paper! My affiliation is fine.

I mostly used windrose for teaching:

![map_zhadang_00_all_year](https://user-images.githubusercontent.com/10050469/39920350-1b186cfa-5517-11e8-8a33-cff05d103740.png)

Thanks for the nice lib!

**Comment by scls19fr** [comment]
Personaly, I'm also using it for teaching purpose... not for research.
@fmaussion Could you post code to overlay bitmap and windrose plot ?
You can even send a PR to add it to `samples` directory.
I like papers with nice pictures! Where is it located?

**Comment by fmaussion** [comment]
@scls19fr shame on me but since this was a one time thing I just used GIMP for the overlay :1st_place_medal: 

If I find time I'd be happy to give this as an exercise for my students ;-), for example as an overlay to a [google static map](http://salem.readthedocs.io/en/latest/auto_examples/plot_googlestatic.html#sphx-glr-auto-examples-plot-googlestatic-py)

**Comment by cqcn1991** [comment]
Great work! I used windrose to plot windrose (as in my https://github.com/cqcn1991/Wind-Speed-Analysis repo), LOL, not really sure to describe how I use it. 

Is there anything I should do or help?

**Comment by ocefpaf** [comment]
@scls19fr thanks for putting this together! I updated my affiliation in https://github.com/python-windrose/windrose/pull/85

BTW, I use `windrose` mostly for teaching and some technical reports every now and then.

**Comment by kilojoules** [comment]
Hey thanks! My name is Julian Quick and I work for the National Renewable Energy Laboratory in Golden, CO. I'm not sure how to where to put the PR so I'm leaving this as a comment.

**Comment by scls19fr** [comment]
You are a contributor of windrose (especially in Dec 2016). Could you tell us in a few words what you are doing / was doing with windrose?

**Comment by kilojoules** [comment]
Yup. I used windrose to make a figure for a [wind plant optimization study](https://www.nrel.gov/docs/fy17osti/68185.pdf). In particular, I added the capability to make wind roses based on the Weibull shape and scale factors.

**Comment by xmnlab** [comment]
@scls19fr great idea about the paper!

My affiliation is good. Actually my bug fix was to help a friend. She was using python-windrose on her thesis.

**Comment by blaylockbk** [comment]
The windrose module is a great tool. Thanks for maintaining it!

My name is Brian Blaylock and I'm at the University of Utah Department of
Atmospheric Sciences. I create wind roses using this Python module for
weather station data from the MesoWest and SynopticLabs API:
https://synopticlabs.org/api/. I have an experimental web page that
generates wind roses on the fly for any station in the MesoWest database
and for any period of record:
http://home.chpc.utah.edu/~u0553130/Brian_Blaylock/cgi-bin/roses.cgi. These
were used to make pollution roses for an ozone study near the Great Salt
Lake (see figure 3,
https://rmets.onlinelibrary.wiley.com/doi/abs/10.1002/asl.680 In Python I
turned on the figure transparency when saving the image and then overlaid
the images in photoshop).

I am currently working with multi-year HRRR model data, and would like to
generate wind roses at any model grid points. To do that, I plan to
generate the histogram table for all model grid points from a three year
archive of HRRR data in advance, and then would need to make the windrose
figures using that histogram table. I would need to modify the windrose
code to make windroses from a historgram table rather than from two arrays
of wind speed and wind direction.


*Brian*

**Comment by scls19fr** [comment]
Thanks @blaylockbk . Your publication clearly shows that we need to fix #43 and that there is also a need for map overlay directly in the library.

**Comment by jsignell** [comment]
My name is Julia Signell and I was using windrose while working at Princeton University on a project tracking lightning strikes during high intensity storms. I was using it to visualize the motion of the storm based on the relative position of the lightning from one strike to the next.

**Comment by LionelR** [comment]
Hi,
I'm the original author of this package and I'm really impressed by it's evolution. This project was started because in my previous job (air pollutant monitoring in France) there was not simple tool to make wind plots in Python, which is IMHO the best language in the world :) With it I wrote technical reports on pollution exposure and wind distributions analyzes, by mixing local pollution measures and meteorologic informations from various sources like Meteo-France. 
scls19fr makes a really great job by continuing the development of this tool.

**Comment by weber-s** [comment]
Woh that's great! I didn't know that such journal as JOSS exist! 

I add my affiliation in a PR: Institut des Géosciences de l'Environnement, Université Grenoble Alpes. 

I use windrose mainly for teaching/data exploration right now. I am in the field of Air Quality. I also may add windrose in one of my side-project: a API and GUI tools for PSCF ([pyPSCF](https://github.com/weber-s/pyPSCF))!
I will certainly also use it in the near future to detect local pollution source.

thanks!

**Comment by scls19fr** [comment]
Could you please have a look at https://github.com/python-windrose/windrose/blob/master/paper/paper.md and edit / submit PR to improve article?

**Comment by fmaussion** [comment]
Hi @scls19fr , thanks! Because of lack of time I just had a quick skim through the paper, it is nice and short! It looks like it could use some English editing, maybe a native speaker will step up and help a little!

**Comment by kilojoules** [comment]
Native english speaker here. I suggested some changes:
https://github.com/python-windrose/windrose/pull/97

Best,
Julian

**Comment by Data-drone** [comment]
I was using windrose to analyse environmental effects on dust dispersion for atmospheric quality control.
